### PR TITLE
fix(loggin.js): add winston patch to fix date problem

### LINKED ISSUE #363: Passport logs printing wrong datetime
Currently, it is printing the wrong datetime in logs:
```
==> /opt/gluu/node/passport/logs/passport.log <==
2021-10-18T16:56:24.149Z [INFO]     Configuration data has been parsed
2021-10-18T16:57:23.933Z [INFO]     getConfiguration. Passport configs received
2021-10-18T16:57:23.934Z [INFO]     Response received
2021-10-18T16:57:23.934Z [INFO]     Configuration data has been parsed
2021-10-18T16:58:23.981Z [INFO]     getConfiguration. Passport configs received
2021-10-18T16:58:23.981Z [INFO]     Response received
2021-10-18T16:58:23.982Z [INFO]     Configuration data has been parsed
2021-10-18T16:59:24.045Z [INFO]     getConfiguration. Passport configs received
2021-10-18T16:59:24.045Z [INFO]     Response received
2021-10-18T16:59:24.045Z [INFO]     Configuration data has been parsed

==> /opt/gluu/jetty/oxauth/logs/oxauth.log <==
2021-10-18 11:11:21,754 INFO  [qtp1383524016-20] [org.gluu.oxauth.auth.Authenticator] (Authenticator.java:279) - Authentication success for Client: '1501.74c01981-202a-4919-aa42-65be5470557a'
2021-10-18 11:16:22,035 INFO  [qtp1383524016-19] [org.gluu.oxauth.auth.Authenticator] (Authenticator.java:279) - Authentication success for Client: '1501.74c01981-202a-4919-aa42-65be5470557a'
2021-10-18 11:21:22,293 INFO  [qtp1383524016-12] [org.gluu.oxauth.auth.Authenticator] (Authenticator.java:279) - Authentication success for Client: '1501.74c01981-202a-4919-aa42-65be5470557a'

```

[winston logger issue](https://github.com/winstonjs/winston/issues/1796)

Environment:
Gluu CE 4.3.0 VM Package
CentOS Server
[support issue](https://support.gluu.org/outages/10026/saml-login-failed-keeps-redirected-to-idp-selector-page/)

### PR DESCRIPTION
**Pull request by kdhttps**
close #363

### CODE REVIEW

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

**Comment by christian-hawk** [comment]
run tests

File: test/logging.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const path = require('path')
+const fs = require('fs').promises
+
+const convertOneDigitToTwo = (number) => {
+  return ('0' + (number)).slice(-2)
+}
+
+describe('logging', () => {
+  it('should log currect datetime once app start', async () => {
+    const passportLogFilePath = path.join(__dirname, '../server/utils/logs/passport.log')
```
Comment (christian-hawk) [inline test/logging.spec.js:13]: Why are we using **app start** as trigger?
Wouldn't it make more sense to trigger a `log2` call and check the date, so we can isolate test?

Comment (kdhttps) [reply test/logging.spec.js:14]: sure agreed, pushed changed, thanks mate!

**Review by christian-hawk — APPROVED** [review]

**Pull request merged**